app/models/form/subsection.rb

```diff
@@ -52,7 +52,7 @@ def answered_questions_count(case_log)
   end
 
   def applicable_questions(case_log)
```

Comment:
It looks like this method is only used by check answers page. Might be worth renaming it to make that clearer?

Reply:
Hmm, it's also used in `status` method by the looks of it, which is used in the `tasklist`

Reply:
In that case does this change break that? (Should maybe have a test for it?). I'm thinking for a section that includes a question like this (hidden from check answers), will the status be set correctly?

Reply:
Which now needs to be fixed, because it shows up as `in progress` even if it's `completed` if postcode is selected as not known. Thanks for flagging!

Reply:
This is now described in separate tickets (CLDC-830,CLDC-831,CLDC-832), so will merge this branch